nilearn/datasets/tests/test_atlas.py

```diff
@@ -417,3 +417,11 @@ def test_fetch_atlas_allen_2011():
         assert_equal(bunch[key], os.path.join(tst.tmpdir, 'allen_rsn_2011', fn))
 
     assert_not_equal(bunch.description, '')
+
+
```

Comment:
This needs to run without internet connection, hence you should use (as in the other tests of this file), the test set up / tear down:

<pre>
 @with_setup(setup_mock, teardown_mock)
 @with_setup(tst.setup_tmpdata, tst.teardown_tmpdata)
</pre>